tests/test_storageclass_invalid.py

```diff
@@ -1,19 +1,95 @@
 import logging
 import pytest
+import os.path
+import yaml
 
 from resources.pvc import PVC
 from tests import helpers
 from ocs import constants, defaults
 from ocs.exceptions import TimeoutExpiredError
 from ocsci.testlib import tier3, ManageTest
+from resources.ocs import OCS
 
 
 logger = logging.getLogger(__name__)
 
 
+@pytest.fixture(
+    params=[pytest.param({
```

Comment:
Can we define the dictionaries being passed to `pytest.param()` outside of the fixture decorator? I think it would make this a lot easier to read.

Reply:
Thank you! Done